migrations/versions/0140_sms_prefix_non_nullable.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
You'll need update this - there is a 0140 already

Reply:
Was only showing up because this was branched off the other branch. There’s no migration in this PR.